django/views/generic/detail.py

```diff
@@ -123,7 +123,10 @@ def get_template_names(self):
         Return a list of template names to be used for the request. May not be
         called if render_to_response() is overridden. Return the following list:
```

Comment:
When I first looked at this without reading the code, I was sure how the bullets were grouped. (A OR B) AND C?  No... A OR (B & C). Something like this would look a little more natural to me:

```
Return a list containing template_name, if set on the value. Otherwise, return a list containing:
* bullet 
* items
```